e2e/help/help.go

```diff
@@ -169,16 +169,55 @@ func (c *ctx) testFailure(t *testing.T) {
 
 }
 
+// Expected first lines of the help command output based on what options is used
+const (
+	helpExpectedOutput     = ""
+	helpHelpExpectedOutput = "Help about any command"
```

Comment:
I understand why you named this, and honestly I can't think of a better name atm, but given the context of the tests I think it makes sense 😄